src/App.jsx

```diff
@@ -1,14 +1,20 @@
 import { useState } from 'react'
 import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
-import './App.css'
+//import './App.css'
+import Header from './components/Header'
+import HairDressers from './HairDressers'
+//import Footer from './Footer'
 
 function App() {
   const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
+    <Header></Header>
+    <HairDressers></HairDressers>
+    
+      {/* <div>
         <a href="https://vitejs.dev" target="_blank">
           <img src={viteLogo} className="logo" alt="Vite logo" />
         </a>
```

Comment:
If you don't use some code here like the vite logo etc, just remove it, do not comment